plugins/pulp_docker/plugins/distributors/publish_steps.py

```diff
@@ -300,12 +359,16 @@ def process_main(self):
         registry = configuration.get_repo_registry_id(self.get_repo(), self.get_config())
         redirect_url = configuration.get_redirect_url(self.get_config(), self.get_repo(), 'v2')
         schema2_data = self.redirect_data[2]
+        manifest_list_data = self.redirect_data['list']
+        manifest_list_amd64 = self.redirect_data['amd64']
 
         redirect_data = {
```

Comment:
I'm surprised this doesn't exist as a separate function or class. If you feel like promoting it to be its own thing, independently testable and documented, that would be an improvement.